tests/Integration/Infrastructure/Controller/Regulation/RegulationDetailControllerTest.php

```diff
@@ -68,6 +68,27 @@ public function testDraftRegulationDetail(): void
         $this->assertSame('/regulations?tab=temporary', $goBackLink->extract(['href'])[0]);
     }
 
+    public function testDraftRegulationWithNewUIDetail(): void
```

Comment:
(Détail) Ici on sort du cas de "locationNew" (qu'on avait déjà débattu), et je continue de penser qu'on devrait éviter d'écrire "new" dans le "nouveau code" et préférer le traiter comme du code "actuel" (puisqu'il a vocation à rester)

```suggestion
    public function testDraftRegulationWithMeasuresDetail(): void
```